common/table.js

```diff
@@ -218,7 +221,14 @@
                     return defer.resolve(true);
                 }).catch(function(err) {
                     if (vm.reference.uri !== uri) {
-                        return defer.resolve(false);
+                        defer.resolve(false);
+                    } else {
```

Comment:
Your else case here ignores the code that is supposed to run outside of it. This was previously refactored to behave like the following:

```
if vm.reference.uri !== uri) {
  return defer.resolve(false)
}
// continue code execution if above condition is not met
vm.hasLoaded = true;
vm.initialized = true;
...
return defer.reject(err);
```